core-bundle/src/Resources/contao/themes/flexible/main.css

```diff
@@ -1727,10 +1727,10 @@ ul.sgallery li {
 /* Misc */
 .ce_gallery ul {
 	overflow:hidden;
```

Comment:
I think we can remove this. It was used to clear the floats of the `li` elements, which are no longer floating now.